sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -384,4 +385,15 @@ class EthService(val scNodeViewHolderRef: ActorRef, val nvtimeout: FiniteDuratio
       }
     }
   }
+
+  @RpcMethod("eth_getForwardTransfers")
```

Comment:
Actually, I think we should call this `zen_getForwardTransfers`.
Moreover, the idea was to have all the `eth_` methods in `EthService` not *all* RPC methods - but that is refactoring for later.

Reply:
so far, let's keep it as is.
Agree to keep the new methods with another prefix.